waspc/cli/Command/Deps.hs

```diff
@@ -0,0 +1,27 @@
+module Command.Deps
+  ( deps,
+  )
+where
+
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import Generator.ServerGenerator as Server (waspNpmDeps)
+import qualified Generator.WebAppGenerator as WebApp (waspNpmDeps)
+import NpmDependency (printDep)
+import qualified Util.Terminal as Term
+
+deps :: Command ()
+deps =
+  liftIO $
+    putStrLn $
+      unlines $
+        [ "",
+          title "WASP DEPENDENCIES",
+          title "Server dependencies:"
+        ]
+          ++ map printDep Server.waspNpmDeps
+          ++ ["", title "Webapp dependencies:"]
+          ++ map printDep WebApp.waspNpmDeps
+
+title :: String -> String
```

Comment:
So this is duplication from the Main.hs.
Question is, do we want it or not?

I think we want to have common style across the CLI, meaning that all titles should look the same.
In that case, this is duplication and we should extract it.
I suggest putting it in cli/Cli/Terminal.hs (doesn't exist yet).

Reply:
Done!